### ISSUE TITLE
Payara 5.201 - JDK 11 - HTTP/2 /FISH-31

### ISSUE THREAD

**Issue by hink084** [payara/Payara#4609]
# Description #
----------
HTTP/2 is not working as expected with Payara 5.201 JDK11.

## Expected Outcome

I would expect to see resources being sent using the HTTP/2 protocol.

## Current Outcome

Resources are being sent using the HTTP/1.1 protocol.

This can be validated using the Google Chrome Developer Tools --> Network Tab --> Protocol Column.

I am noticing the following WARNING in the Payara log file:
```
[2020-04-03T12:56:07.769-0500] [Payara 5.201] [WARNING] [] [org.glassfish.grizzly.http2.Http2AddOn] [tid: _ThreadID=19 _ThreadName=RunLevelControllerThread-1585936563404] [timeMillis: 1585936567769] [levelValue: 900] [[
  TLS ALPN (Application-Layer Protocol Negotiation) support is not available. HTTP/2 support will not be enabled.]]
```

## Steps to reproduce (Only for bug reports) 

1. Start the domain - ./asadmin start-domain
2. Login to the DAS - https://your.server.com:4848
3. Navigate to: Configurations --> server-config --> Network Config --> Protocols --> http-listener-2 --> HTTP Tab.
4. HTTP/2 --> Checked
5. HTTP/2 Push Enabled --> Checked

My full protocol configuration is as follows:
```
<protocol name="http-listener-2" security-enabled="true">
    <http xpowered-by="false" http2-enabled="true" http2-push-enabled="true" timeout-seconds="3600" server-header="false" compressable-mime-type="text/html,text/xml,text/plain,text/css,text/javascript,application/javascript" max-connections="250" compression="on" default-virtual-server="server">
        <file-cache max-age-seconds="180" enabled="true"></file-cache>
    </http>
    <ssl classname="com.sun.enterprise.security.ssl.GlassfishSSLImpl" cert-nickname="mycert"></ssl>
</protocol>
```

## Environment ##

- **Payara Version**: 5.201 (Build 512)
- **Edition**: Full
- **JDK Version**: OpenJDK 11.0.6+10
- **Operating System**: Oracle Linux 7.7
- **Database**: N/A

**Comment by fturizo** [comment]
@hink084, this is a known issue and has already been reported by the community on #2625. Since this issue is a duplicate of that one, we'll proceed to close it. Please, follow the resolution of that ticket when a potential fix is ready.

**Comment by OndroMih** [comment]
This is actually a different issue than #2625. We already know about this, it's raised as our internal issue CUSTCOM-259 but there's no github issue for this yet, so I'll reopen this one.

**Comment by fturizo** [comment]
Issue solved and released as part of 5.2020.3 with the ID `FISH-31`
